automation/dependDockerfile/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM quay.io/openshift/origin-jenkins-agent-base:v4.0.0
```

Comment:
you are not using the "cleanDockerfile", then why do you need it?

Reply:
I think that we used this Dockerfile to create the containers in Jenkins 'ssh_slave' nodes.

Reply:
so why do we need two Dockerfiles?

Reply:
Ok, I'll remove it